cinder/README.md

```diff
@@ -16,49 +16,52 @@ The Dell EMC Cinder container image contains following RPM packages:
 
 ### Prerequisites
 
-- Red Hat OpenStack Platform 15.
+- Red Hat OpenStack Platform 16.
 - VNX with Block version 5.32 or above.
 
 ### Steps
 
-#### Prepare Dell EMC container
+#### 1. Prepare Dell EMC container
 
 The formal Dell EMC container image is published to [Red Hat Container Catalog](https://access.redhat.com/containers/)
 
 Red Hat OpenStack Platform supports remote registry and local registry for overcloud deployment. In this document, we only introduce local registry.
 
-> in below examples, 192.168.139.1:8787 acts as a local registry.
+> In below examples, 192.168.139.1:8787 acts as a local registry.
+
+**Notes:** We will not introduce how to setup local registry with docker-registry or docker-distribution. In RHOSP16,  undercloud also acts as a local registry, you could push Dell EMC container images to it, and pull these images when deploying overcloud, please refer Red Hat documents.
```

Comment:
please refer to Red Hat documents.
If there is link pointing to the documents, that will be better.

Reply:
Done.